roles/contiv_network/vars/main.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
this is minor but let's use the `defaults` directory (instead of `vars`) to introduce these variables. 

We depend on ability to override some of the variables in the `installer` scripts and `cluster-mgr` heavily so choice of defining variables matters. `defaults` are preferred as they hold least precedence and allow ease of overriding from command line, inventory variables etc.